sdk/rust/src/lib.rs

```diff
@@ -11,6 +11,13 @@ pub mod key_value;
 /// Exports the procedural macros for writing handlers for Spin components.
 pub use spin_macro::*;
 
+#[export_name = "spin-sdk-version-0-10-pre0"]
```

Comment:
Would it be worth doing something like this?

```suggestion
#[export_name = concat!("spin-sdk-version-",
                        env!("CARGO_PKG_VERSION_MAJOR"),
                        "-",
                        env!("CARGO_PKG_VERSION_MINOR"),
                    )]
```

Reply:
Yup, done.  Plus now we have `spin-sdk-commit-$COMMIT` as well.